src/js/plugins/plugin.text_selection.js

```diff
@@ -212,7 +237,7 @@ export class BookreaderWithTextSelection extends BookReader {
   _createPageContainer(index, styles = {}) {
     const $container = super._createPageContainer(index, styles);
     // Disable if thumb mode; it's too janky
-    if(this.enableTextSelection && this.mode != this.constModeThumb){
+    if(this.textSelectionPlugin && this.mode != this.constModeThumb){
       this.textSelectionPlugin.createTextLayer(index, $container);
```

Comment:
Can you use an optional here too, ie:
```
if(this.mode != this.constModeThumb){
  this.textSelectionPlugin?.createTextLayer(index, $container); 
}
```

Reply:
Ah, good idea! I thought this one stood out :D

Reply:
Fixed